packages/cli/resources/change-report/findings/details.hbs

```diff
@@ -53,24 +53,25 @@
 {{/inline}}
 {{#if (length findingDiff.newFindings findingDiff.resolvedFindings)}}
 <h2 id="{{ metadata.anchor }}">{{ metadata.title }}</h2>
+
```

Comment:
This space here is actually a fix. Without the space, the `###` markdown on the next line isn't rendered.